src/plugins/orangehrmCorporateDirectoryPlugin/test/orangehrmCorporateDirectoryPluginAllTests.php

```diff
@@ -5,21 +5,19 @@
  * and open the template in the editor.
  */
 
-class orangehrmCorporateDirectoryPluginAllTests {
-    public static function suite() {
+use PHPUnit\Framework\TestSuite;
 
-        $suite = new PHPUnit\Framework\TestSuite('orangehrmCorporateDirectoryPlugin');
+class orangehrmCorporateDirectoryPluginAllTests
```

Comment:
delete this file. not using anymore with 5.x

Reply:
Done